apps/fabric-website/src/components/Nav/Nav.tsx

```diff
@@ -24,9 +25,11 @@ export class Nav extends React.Component<INavProps, INavState> {
       : null;
 
     return (
-      <nav className={ styles.nav } role='navigation'>
-        { links }
-      </nav>
+      <FocusZone ref='focusZone' direction={ FocusZoneDirection.vertical } role='menubar' >
```

Comment:
I pulled this implementation over from NavBar. I can remove the ref. As for the role, which should we use?